atomics/T1543.004/T1543.004.yaml

```diff
@@ -34,3 +34,36 @@ atomic_tests:
       sudo launchctl unload /Library/LaunchDaemons/#{plist_filename}
       sudo rm /Library/LaunchDaemons/#{plist_filename}
       sudo rm /tmp/T1543_004_atomicredteam.txt
+- name: Launch Daemon - Users Directory
+  auto_generated_guid:
+  description: |
+    Utilize LaunchDaemon in /Users directory  to touch temporary file in /tmp
+  supported_platforms:
+  - macos
```

Comment:
Apologies for the late response, I was on time off myself. I think that could be a valid solution as well, and perhaps more useful for some strange edge cases, but launch daemons are only seen and ran if placed in one of these two directories. Putting it anywhere else wouldn't quite be fair to say is emulating any sort of attack because it would never be run. Could be done, but for the sake of cutting out user input I think this would be fine for now.